source/loader/layers/validation/ur_leak_check.hpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+#ifndef UR_LEAK_CHECK_H
+#define UR_LEAK_CHECK_H 1
+
+#include "backtrace.hpp"
+#include "ur_validation_layer.hpp"
+
+#include <mutex>
+#include <unordered_map>
+#include <utility>
+
+#define MAX_BACKTRACE_FRAMES 64
+
+namespace validation_layer {
+struct RefCountContext {
```

Comment:
nit: An empty line at the beginning (and before the end) of the namespace scope for better readability, just as in other files.

Reply:
Done.